bundles/org.openhab.ui/web/src/components/cards/glance/location/measurement-badge.vue

```diff
@@ -73,14 +73,14 @@ export default {
       }
     },
     map () {
-      return this.query.map((item) => this.store[item.name].state).filter((state) => Number.isFinite(Number.parseFloat(state.split(' ')[0])))
+      return this.query.map((item) => this.store[item.name].state).filter((state) => Number.isFinite(Number.parseFloat(state)))
```

Comment:
Can you explain why these `.split(' ')|0]` were removed? They seemed to be useful when you had math operations to perform on quantity types (represented as strings with the unit in the API, you you can't perform e.g. averages with them).

Reply:
```If parseFloat encounters a character other than a plus sign (+), minus sign (- U+002D HYPHEN-MINUS), numeral (0–9), decimal point (.), or exponent (e or E), it returns the value up to that character, ignoring the invalid character and characters following it.```

So IMHO it's not needed ... from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat

```
The following examples all return 3.14:

parseFloat(3.14);
parseFloat('3.14');
parseFloat('  3.14  ');
parseFloat('314e-2');
parseFloat('0.0314E+2');
parseFloat('3.14some non-digit characters');
```